pipelines/frontend-pipeline.yml

```diff
@@ -0,0 +1,83 @@
+# Starter pipeline
+# Start with a minimal pipeline that you can customize to build and deploy your code.
+# Add steps that build, run tests, deploy, and more:
+# https://aka.ms/yaml
+
+trigger:
+  branches:
+    include:
+      - master
+  paths:
+    include:
+      - frontend/*
+      - pipelines/frontend-pipelines.yml
+    
+  pr:
+    autoCancel: true   
+    branches:
+      include:
+        - master
+    paths:
+      include:
+        - frontend/*
+        - pipelines/frontend-pipelines.yml
+
+pool:
+  vmImage: 'ubuntu-latest'
+
+variables:
+  - group: prodoh-urlist-variable-group
+
+stages:
+  - stage: BuildStage
+    displayName: Build VUE Frontend
+    jobs:
+      - job: BuildJob
+        displayName: Build Test and Publish
```

Comment:
Based on this current naming scheme of the stage and job, the display name for the job will look like this. 'Build VUE Frontend Build Test and Publish'. Its weird and doesn`t show up like that in my personal AzDo but it does show in Prod account.  I am changing the name scheme for API and I would recommend you to change it from FE as well.

Suggestion:
Stage display name: Build
Job display name: Frontend